### ISSUE TITLE
Developer Advocate

### ISSUE THREAD

**Issue by SachaMorard** [edgee-cloud/careers#4]
# Join Edgee as a Developer Advocate 

- Location: Remote
- Type: Full-time

Are you passionate about sharing knowledge, building communities, and driving innovation in the tech industry? Do you have a knack for simplifying complex technical concepts and inspiring others to learn? We're looking for a talented Developer Advocate to join our team at Edgee and help shape the future of Edge Computing.

## Your Mission

As a Developer Advocate at Edgee, you'll play a key role in driving the adoption of our Edge Computing technology by educating, supporting, and empowering the developer community. Your primary focus will be on creating high-quality developer documentation, engaging with the open-source community, and developing technical content that showcases the power of Edgee.

## Key Responsibilities

- Develop and lead technical workshops, group learning sessions, and other technical content to accelerate the adoption of Edgee.
- Create engaging developer documentation and content that simplifies complex technical concepts
- Engage with the open-source community, provide support, and enable growth
- Educate the community on topics related to WebAssembly (WASM) and Edge Computing

## Requirements

- 5+ years of professional experience as a developer
- 3+ years of experience in storytelling and content creation (blogs, conferences, leading workshops, live streaming, contributing to open source, participating in community events)
- Proficiency in Frontend development
- Fluent English and French

## Nice to Have

- Experience with open-source contribution and community management
- Event hosting and organization skills
- Familiarity with WebAssembly (WASM)

## Why Edgee

**Innovative Work:** At Edgee, you’ll work at the forefront of edge computing, shaping the future of technology.
**Inclusive Culture:** We celebrate diversity and are committed to creating an inclusive environment for all employees.
**Professional Growth:** Continuous learning is part of our DNA. We support your professional development and growth every step of the way.
**Competitive Benefits:** We offer competitive salaries, health benefits, and a flexible remote work environment.

And guess what? Edgee is a very early-stage start-up, so everything needs to be built and the sky is the limit.

### 🙌🏽 Execute
Execution is our daily obsession. We know our ability to execute is our first success factor and will make a difference in the short and long run.

### ✅ Do What We Say and Say What We Do
No-frills, just impact. We focus on having things achieved, openly share failures, experiences, and challenges, and celebrate achievements. We are transparent and have the self-confidence to act like that.

### 🫱🏼‍🫲🏽 Gather 
We love to share, meet, and communicate transparently and spontaneously about questions, challenges, fears, or simply our day. We do it on a daily basis internally as well as with partners and customers.

### 🚀 Engage
We believe in people and trust them to share and engage the best of them. In return, we offer them a high level of freedom and autonomy in their job. 

## Benefits: 

Health insurance + Dental insurance + Pension/401k + Parental leave + Fully remote work anywhere in Europe or North America + unlimited holidays. What else?

## Hiring Process

All meetings are conducted remotely via Google Meet and are 30mn to 1 hour long maximum. 

5 steps and you can be on board

**1. Pre-screening:**
Send your CV and three short bullet points explaining why you might be a good fit for this role to recruitme@edgee.cloud.

**2. Screening Interview:**
If your profile fits, we’ll invite you for a 30-minute video call to learn more about your background and motivations and to answer your key questions with Gilles

**3. Technical Fit Interview with Sacha:**
You’ll meet with [Sacha Morard](https://github.com/sachamorard), for a 45 to 60-minute pair programming meeting. You’ll work together on a problem closely related to the scope of the team.

**4. Team Fit Interview with Gilles:**
You’ll meet with [Gilles Raymond](https://www.linkedin.com/in/gillesraymondmobile/), for a 45 to 60-minute meeting. We want to know you better, and also for you to know us better by asking questions. 

**5. Reference Check:**
In parallel with the final interviews, we’ll conduct a reference check with two people of your choosing who you have worked with or are currently working with.

**5. Offer**
We run a last call to answer your remaining questions and build together a proposal.

The duration of the full process is less than 30 days.
